### ISSUE TITLE
BasicFloodingProtection considers redactions and triggers especially on Nheko automatic message removal

### ISSUE THREAD

**Issue by Mikaela** [the-draupnir-project/Draupnir#187]
In addition to the Draupnir4All instance in my rooms wanting to ban me (https://github.com/the-draupnir-project/Draupnir/issues/161), I am told my Beeper account got banned from LibreWolf (which I don't moderate) and I wonder when will I be banned from FluffyChat (where I do moderate) and it's kind of getting embarassing.

I wonder if there may also be a legal issue with GDPR and equivalent legislation in regions with strong privacy protections (while trying to not enter the rabbit hole of whether banning preventing right to be forgotten is a legal issue).

I think everyone has seen the issue already, but I wish to bumb it as it's just going to become more common. If someone has avoided hearing of the Nheko feature,

- https://aminda.eu/matrix/#how-can-i-remove-my-messages-automatically-like-on-signal-whatsapp-telegram-and-everything-else

I know https://github.com/matrix-org/matrix-spec-proposals/pull/4084 is a thing, but it won't help until everyone has upgraded to room version containing it, which again may be 12 and I believe the current default recommended one is 10.

**Comment by ninchuka** [comment]
yeah just had this with a user in privacyguides, I was confused since I didnt see any messages on nheko (so no view all events) and didnt realise till they asked to be unbanned
